draft-ietf-tls-tls13.md

```diff
@@ -967,12 +967,21 @@ establishment, thus providing forward secrecy.
 
 ## Zero-RTT Data
 
-When resuming via a PSK with an appropriate ticket (i.e., one with
-the "early_data_info" extension), clients can also send data on their first
-flight ("early data"). This data is encrypted solely under keys
-derived using the first offered PSK as the static secret.  As shown in
-{{tls-0-rtt}}, the Zero-RTT data is just added to the 1-RTT handshake
-in the first flight. The rest of the handshake uses the same messages
+With the Zero-RTT mode clients can send data on their first flight
+("early data") whereby the client uses a PSK either obtained 
+out-of-band or as a ticket (i.e., one with the "early_data_info" 
+extension) from an earlier exchange to authenticate to the server. 
+
+When clients use a PSK obtained out-of-band then the following 
+information MUST be provisioned to both parties: 
+  * PSK identity, 
+  * ciphersuite for use with this PSK, 
+  * key exchange and authentication modes this PSK is allowed to be used with,  
+  * the Application-Layer Protocol Negotiation (ALPN) label(s), and 
+  * optionally the Server Name Indication (SNI). 
```

Comment:
I suspect that the SNI line needs to be "the Server Name Indication (SNI), if any is to be used"